differ/differ.go

```diff
@@ -700,14 +840,22 @@ func closeStorage(storage *DBStorage) {
 	}
 }
 
-func closeNotifier() {
-	err := notifier.Close()
+func closeKafkaNotifier() {
+	err := kafkaNotifier.Close()
 	if err != nil {
 		log.Err(err).Msg(operationFailedMessage)
 		os.Exit(ExitStatusKafkaConnectionNotClosedError)
```

Comment:
if it exit, no other close() will be called

Reply:
True, added `defer` to `os.Exit` calls.